qase-javascript-commons/changelog.md

```diff
@@ -1,3 +1,20 @@
+# qase-javascript-commons@2.0.9
+
+## What's new
+
+Added new configuration option `rootSuite` to specify a root suite.
+This option is available in the config file and the `QASE_ROOT_SUITE` env variable
```

Comment:
Does the root suite apply to local reports? If not, it should probably belong to the testops config sections and be named `QASE_TESTOPS_ROOT_SUITE`.

Reply:
yes, the root suite applies to both reporters: testops and local